packages/atomic/src/components/search/atomic-generated-answer/typing-loader.tsx

```diff
@@ -0,0 +1,18 @@
+import {FunctionalComponent, h} from '@stencil/core';
+
+interface TypingLoaderProps {
+  loadingLabel: string;
+}
+
+export const TypingLoader: FunctionalComponent<TypingLoaderProps> = (props) => (
+  <div>
+    <div class="typing-indicator">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
```

Comment:
Since this is purely decorative and there's already a loading label underneath, this should probably have `aria-hidden="true"` to hide it from screen readers.